templates/prescription/general_edit.html

```diff
@@ -256,10 +280,32 @@
 {$ENDING_JAVASCRIPT}
 </script>
 </form>
+<script type="text/javascript" src="{$WEBROOT}/public/assets/jquery-min-1-9-1/index.js"></script>
+<script type="text/javascript" src="{$WEBROOT}/library/js/jquery-ui.min.js"></script>
```

Comment:
removed those line already and I put it in because I was not sure if there is a diff in those file from version to version.